src/gsAssembler/gsExpressions.h

```diff
@@ -2844,6 +2868,7 @@ GISMO_SHORTCUT_VAR_EXPRESSION(  div, jac(u).trace() )
 GISMO_SHORTCUT_MAP_EXPRESSION(unv, nv(G).normalized()   ) //(!) bug + mem. leak
 
 GISMO_SHORTCUT_PHY_EXPRESSION(igrad, grad(u)*jac(G).ginv() ) // transpose() problem ??
+GISMO_SHORTCUT_VAR_EXPRESSION(igrad, grad(u) ) // u is presumed to be defined over G
```

Comment:
Does this now mean igrad == grad ?

Reply:
yes, user is aware of where the function is defined..

Reply:
i am still not yet sure how to work with the assembler in this case.

Reply:
its 2 different commands. 
igrad(f,G) is the isogeometric gradient of paremeter-defined function
igrad(f) is the isogeometric gradient of function defined on physical coordinates.
I don't know a better way to express this..